ui/app/components/pricing-metrics-dates.js

```diff
@@ -69,7 +69,7 @@ export default Component.extend({
     }
   }),
 
-  showResultsWarning: computed('resultStart', 'resultEnd', function() {
+  showResultsWarning: computed('queryEnd', 'queryStart', 'resultEnd', 'resultStart', function() {
```

Comment:
This might be unintended behavior: when you change the query and it doesn't match the result, one would _not_ expect the warning to show up before query is processed